tests/kuttl/tests/pre-caching-partial-complete/08-errors.yaml

```diff
@@ -0,0 +1,109 @@
+# MCVs and MCAs for failed clusters should be cleaned up after CGU completes
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: view-precache-cluster-role-binding
+  namespace: spoke2
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: view-precache-namespace
+  namespace: spoke2
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: view-precache-service-acct
+  namespace: spoke2
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: view-precache-spec-configmap
+  namespace: spoke2
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: view-precache-cluster-role-binding
+  namespace: spoke5
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: view-precache-namespace
+  namespace: spoke5
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: view-precache-service-acct
+  namespace: spoke5
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: view-precache-spec-configmap
+  namespace: spoke5
+---
+apiVersion: view.open-cluster-management.io/v1beta1
```

Comment:
If possible, it will be nice to have all the MCVs for one cluster grouped together.

Reply:
Updated